### DISCUSSION TITLE
I'd love to have separate repos for each app

### DISCUSSION THREAD

**Discussion by pablo03v** [ente-io/ente#1784] [General]
I'm getting error message from Obtainium that the wrong app id had been detected. It would be cool if you could provide separate release channels for each app <3

**Comment by ua741** [comment]
In Obtainium, you should be able to add filter for each app separately
Ref: https://github.com/ente-io/ente/discussions/1087#discussioncomment-8775684

**Reply by pablo03v** [reply]
Thanks but I simply don't know regex and have no clue on how to set it up.

**Reply by xlrshubham** [reply]
Then learn <3

**Reply by pablo03v** [reply]
> Then learn <3

💀
